src/docs/PML/PMLUserGuide.adoc

```diff
@@ -797,23 +797,33 @@ image::MAAS_ML_Delete_Service.PNG[align=center]
 
 There is also one more action that can be executed from Service Automation Portal which is:
 
-- *Update_MaaS_ML_Parameters*: This action enables you to update the variables values associated to the MaaS_ML instance according to your new preferences.
+- *Update_MaaS_ML_Parameters*: This action enables you to update the variables values associated
+to the MaaS_ML instance according to your new preferences.
 
-==== Audit and Traceability
-To access the Audit and Traceability page, click on the endpoint under the Endpoint list.
+==== MaaS_ML Analytics
+When the MaaS_ML service instance is running, the user is able to access the MaaS_ML Analytics
+page by clicking on the instance's endpoint which contains 4 tabs:
 
-In this page, you are able to check the different values of the model variables.
-You can also track the traceability information of the token during several date/time(s) as shown in the below image.
+** Audit and Traceability
+** Dataset Analytics
+** Data Drift Analytics
+** Predictions Preview
+
+===== Audit and Traceability
+
+
+When clicking on the endpoint, the user is redirected to a 4 tabs webpage. By default, the Audit and
+Traceability tab opens in which the user can check the different chosen values of the MaaS_ML instance
+variables. In addition, the traceability information and warnings are listed in a table where each row represents
```

Comment:
each row represents
the initialization, deployment, prediction etc... information 
to refactor like infrormation about the initi.., dep, pred, etc